sql/src/main/java/org/apache/druid/sql/AbstractStatement.java

```diff
@@ -165,12 +180,12 @@ protected PlannerResult plan(DruidPlanner planner)
    */
   public Set<ResourceAction> resources()
   {
-    return queryResource;
+    return cancelationResourceActions;
```

Comment:
cancellation (it's the only form with two Ls according to the most-specific American English technical style guide I could find: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/cancel-canceled-canceling-cancellation; and a variety of dictionaries including MW.)